inc/namespace.php

```diff
@@ -80,7 +80,23 @@ function merge_config_settings( array $config, array $overrides ) : array {
 			// Merge module settings together.
 			case 'modules':
 				foreach ( $value as $module => $settings ) {
-					$config[ $key ][ $module ] = array_merge( $config[ $key ][ $module ] ?? [], $settings );
+					// Check config settings syntax is valid.
```

Comment:
I think this comment would make sense just before the `if` statement below it. Here we're just assigning the value to a variable. Minor thing really but makes it a little clearer.

Reply:
Ah, I did think about that :) changing...